### ISSUE TITLE
code conduct ?

### ISSUE THREAD

**Issue by retouching** [Swiiz/TntRun#1]
Why do you dev your plugin in the main class ? I think, you must structure your code for you and us for a better comprehension of your project and code. So a refactoring of your code must happening !

**Comment by Swiiz** [comment]
That is true, but this was a quick project so I didn't take time to make it really clean but if you are interested in the projet feel free to make a pull request and refactor the code. 👍

**Comment by Swiiz** [comment]
Also there is some datastructure optimization to do if you want to make it clean
